strings/naive_string_search.py

```diff
@@ -11,34 +11,33 @@
 """
 
 
-def naive_pattern_search(main_string: str, pattern: str) -> list:
+def naivePatternSearch(mainString: str, pattern: str) -> list:
```

Comment:
This change is the opposite of what is recommend in CONTRIBUTINg.md.  In this case, you could just get away with `s` as the parameter name.

Reply:
I am afraid I don't understand :(

Reply:
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Reply:
`def naive_pattern_search(main_string: str, pattern: str) -> list:`

Are these the changes you asked me to undo? :smiley:

Reply:
The type hints are all good.